CHANGES.md

```diff
@@ -1,3 +1,60 @@
+### Unreleased
+
+#### Parser (2021-01-??)
+
+Replace Earley by Menhir, Pratter and Sedlex
+
+**Syntax modifications:**
+
+- The semi colon as termination token has been added. Hence, a symbol is
+  declared with symbol `foo: bar;`.
+
+- The syntax `λx y z: nat, ...` is not authorised anymore, but 
+  `λ(x y z: nat), ...` is, and `λ x : N, t` is as well.
+
+- Unification rules syntax has changed: the left hand side must be enclosed in
+  `begin ... end`, so
+  ```
+  set unif_rule $x + $y ≡ 0 ↪ $x ≡ 0; $y ≡ 0
+  ```
+  becomes
+  ```
+  set unif_rule $x + $y ≡ 0 ↪ begin $x ≡ 0; $y ≡ 0 end
+  ```
+- `set declared "?"` is not needed anymore,
```

Comment:
has been removed since a large class of unicode characters is now allowed for identifiers